js/spreadsheet.js

```diff
@@ -861,16 +825,32 @@ var spreadsheet = function(options) {
     return entries;
   };
 
-  function validateJsonString(str) {
-    var validatedString;
-
+  function getColumnValue(str) {
     try {
-      validatedString = jsonObjRegExp.test(str) ? JSON.parse(str) : str;
+      var parsedResult = JSON.parse(str);
+
+      if (typeof parsedResult === 'object') {
+        return parsedResult;
+      }
+
+      return getString(str);
     } catch (e) {
-      validatedString = str;
+      return getString(str);
     }
+  }
 
-    return validatedString;
+  function getString(str) {
+    try {
```

Comment:
Why is this part necessary? What's the problem scenario if we don't use this in the `getColumnValue` function?